{
  "name": "test_apply_data[expect0]",
  "status": "broken",
  "statusDetails": {
    "message": "selenium.common.exceptions.TimeoutException: Message:",
    "trace": "self = <scripts.test02_apply_data.TestApplyData object at 0x0000022B78B46B48>\nexpect = ('请先完成实名认证后再操作',)\n\n    @pytest.mark.parametrize(\"expect\", read_yaml(\"apply_data.yaml\"))\n    def test_apply_data(self, expect):\n    \n        # 调用发布文章业务方法\n>       self.applydata.page_apply_data()\n\ntest02_apply_data.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\page\\page_apply_data.py:59: in page_apply_data\n    self.page_click_check_box()\n..\\page\\page_apply_data.py:38: in page_click_check_box\n    self.base_click(page.readed_check_box)\n..\\base\\base.py:49: in base_click\n    self.base_find(loc).click()\n..\\base\\base.py:27: in base_find\n    .until(lambda x: x.find_element(*loc)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"55c13d9c9d6b90f021dd987160f8e1bb\")>\nmethod = <function Base.base_find.<locals>.<lambda> at 0x0000022B78B3FEE8>\nmessage = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nC:\\Python37\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"
  },
  "attachments": [
    {
      "name": "log",
      "source": "5a89face-7a45-4fa8-aef5-dbd8dc611a1d-attachment.txt",
      "type": "text/plain"
    }
  ],
  "parameters": [
    {
      "name": "expect",
      "value": "('请先完成实名认证后再操作',)"
    }
  ],
  "start": 1615644986047,
  "stop": 1615645008772,
  "uuid": "e0a95a88-80f3-4261-b924-e03746005989",
  "historyId": "7b8b5bd64e078c795fdd9a1f086d2dd1",
  "testCaseId": "9dda7dd57086368af497cd4f63b7fe80",
  "fullName": "scripts.test02_apply_data.TestApplyData#test_apply_data",
  "labels": [
    {
      "name": "parentSuite",
      "value": "scripts"
    },
    {
      "name": "suite",
      "value": "test02_apply_data"
    },
    {
      "name": "subSuite",
      "value": "TestApplyData"
    },
    {
      "name": "host",
      "value": "LAPTOP-O96LCF4D"
    },
    {
      "name": "thread",
      "value": "9200-MainThread"
    },
    {
      "name": "framework",
      "value": "pytest"
    },
    {
      "name": "language",
      "value": "cpython3"
    },
    {
      "name": "package",
      "value": "scripts.test02_apply_data"
    }
  ]
}